fe/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -639,6 +639,11 @@ private void handleInsertStmt() throws Exception {
                 return;
             }
 
+            if (insertStmt.getQueryStmt() != null && (coord.getCommitInfos() == null || coord.getCommitInfos().isEmpty())) {
```

Comment:
Because we have already begin a transaction for this insert statement, if you return here now, it will become a pending transaction. 
I think that you should abort this transaction before return to user

Reply:
👌，I will check

Reply:
> Because we have already begin a transaction for this insert statement, if you return here now, it will become a pending transaction.
> I think that you should abort this transaction before return to user

add abort operation;
sry ,I habitually performed a git rebase operation